JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -219,17 +219,25 @@ private fun BulletParagraph(
     paragraphStyle: ParagraphStyle
 ) {
     Row {
-        Box(
-            modifier = LayoutSize(8.dp, 8.dp),
-            backgroundColor = contentColor(),
-            shape = CircleShape
-        ) {
-            // empty box
+        with(DensityAmbient.current) {
+            // this box is acting as a character, so it's sized with font scaling (sp)
+            Box(
+                    modifier = Modifier
```

Comment:
Formatting is a bit weird here, AS might've gone crazy

Reply:
Good catch!